docs/api/content-tracing.md

```diff
@@ -84,9 +84,19 @@ temporary file. The actual file path will be passed to `callback` if it's not
 ### `contentTracing.getTraceBufferUsage(callback)`
 
 * `callback` Function
-  * `value` Number
-  * `percentage` Number
+  * Object
+    * `value` Number
+    * `percentage` Number
```

Comment:
This is a breaking change, unfortunately, and goes against the project's policy of publicly announcing this at least one major version ahead of time.

We should either add *another* `deprecate()` helper to explicitly address this, or to get a breaking change announcement into the 5.0 betas so that this breaking change can land in 6.0.